main.go

```diff
@@ -77,13 +84,22 @@ func main() {
 				"choices=%v", []string{persistenceModelDirectUnencrypted, persistenceModelDirectEncrypted, persistenceModelNone}))
 	flag.IntVar(&vdsOptions.MaxConcurrentReconciles, "max-concurrent-reconciles-vds", 100,
 		"Maximum number of concurrent reconciles for the VaultDynamicSecrets controller.")
-	flag.BoolVar(&finalizerCleanup, "finalizer-cleanup", false, "Remove finalizers from all CRs in preparation for shutdown.")
+	flag.BoolVar(&preDeleteHook, "pre-delete-hook", false, "Run as helm pre-delete hook")
+	flag.BoolVar(&revokeVaultTokensOnUninstall, "revoke-vault-tokens-on-uninstall", false,
+		"Revoke all cached Vault client tokens on Helm uninstall.")
+	flag.BoolVar(&pruneVaultTokensOnUninstall, "prune-vault-tokens-on-uninstall", false,
+		"Prune all Vault client tokens in storage on Helm uninstall.")
```

Comment:
I wonder if we collapse these two flags into one? Similar to the way that we do for configuring the storage cache.

Reply:
Do you mean something like `VaultTokensCleanUpModelOnUninstall` with values `revoke`, `prune`, `revoke-and-prune`?